packages/plugin-credentials/src/index.js

```diff
@@ -12,7 +12,9 @@ import config from './config';
 import AdvancedAuthInterceptor from './interceptor';
 
 const proxies = [
-  `canAuthorize`
+  `canAuthorize`,
+  `isAuthenticating`,
+  `queryStringProcessed`
```

Comment:
Needed to be proxied to let the web client use it as a notification to whether or not display the login page.